server/modes-config.ts

```diff
@@ -0,0 +1,17 @@
+import { ModesConfig } from './@types/mode-config'
+
+// eslint-disable-next-line import/prefer-default-export
+export const modesConfig: ModesConfig = {
```

Comment:
I'm wondering if we could do an `Object.freeze` here to prevent this configuration being altered inadvertently. Another way would be exposing config through a getter function, i.e. `getConfigFor(mode)`